src/modules/sensors/failure_detector_HITL/failure_detector_HITL.cpp

```diff
@@ -0,0 +1,227 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2012-2024 PX4 Development Team. All rights reserved.
```

Comment:
We can use also "Copyright (c) 2024 Technology Innovation Institute. All rights reserved." Not that it makes much legal difference actually in most countries, but still.

Reply:
Thank you! I have missed this point.